src/utils/rooms.ts

```diff
@@ -493,7 +483,6 @@ export const formatDate = (date: Date) => {
 export type ValidAcronym = "MIA" | "MIB" | "CRG" | "LCF" | "PCL" | "MNI"
 
 export type GlobalRoomListInterface = {
-  expireAt?: string
   searchDate?: string
 } & Record<ValidAcronym, Room[]>
 
```

Comment:
Actually penso si possa eliminare anche la searchDate, alla fine veniva usata per fare il caching in maniera corretta, ma siccome lo implementeremo dentro l'http client penso che da qui si possa togliere.